packages/components/psammead-embed-error/src/index.stories.jsx

```diff
@@ -1,44 +1,51 @@
 import React from 'react';
-import styled from 'styled-components';
 import { storiesOf } from '@storybook/react';
-import { withKnobs, text } from '@storybook/addon-knobs';
+import { withKnobs, text, boolean } from '@storybook/addon-knobs';
 import { withServicesKnob } from '@bbc/psammead-storybook-helpers';
 import notes from '../README.md';
 import EmbedError from './index';
 
-const Container = styled.div`
-  height: calc(100vw * (9 / 16));
-  width: 100vw;
-`;
-
 storiesOf('Components|EmbedError', module)
   .addDecorator(withKnobs)
   .addDecorator(withServicesKnob())
   .add(
     'default',
     ({ service }) => (
-      <Container>
-        <EmbedError
-          service={service}
-          message={text(
-            'message',
-            "Sorry, we're unable to bring you this media right now.",
-          )}
-        />
-      </Container>
+      <EmbedError
+        service={service}
+        message={text(
+          'message',
+          "Sorry, we can't display this part of the story on this lightweight mobile page.",
+        )}
+        link={{
+          text: text(
+            'link text',
+            'View the full version of the page to see all the content.',
+          ),
+          href: '#',
+        }}
+        fillViewport={boolean('fillViewport', false)}
+      />
     ),
     { notes, knobs: { escapeHTML: false } },
   )
   .add(
-    'fill page',
+    'with link',
```

Comment:
Is there a difference between the 'default' and 'with link' stories?

Reply:
Not any more - good catch.
I think @FK78's on to this!

Reply:
Done ✅